runner/runners/invoke.go

```diff
@@ -113,9 +106,7 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 	}
 
 	go func() {
-		//FIXME(jschiller): allow aborts/timeouts to cancel the checkout process.
 		if cmd.SnapshotID == "" {
-			//TODO: we don't want this logic to live here, these decisions should be made at a higher level.
```

Comment:
do we really want to remove this TODO- it seems to be a reasonable comment to me.

Reply:
I don't feel these TODOs about design level decisions add anything. No one is going to notice this comment and then implement its suggestion, and todo's are not an issue tracking system. IMO todos should not be checked into master branch, but I found this one to be particularly useless

Reply:
I agree and disagree. TODO's are useful to document things that are unintuitive, behaviorally or semantically, yet appear to work, and therefore don't warrant polluting jira. In this case, it's unlikely that anyone will ever be impacted, but you *did* notice the TODO and have the option of fixing it or deleting it while you're here.